deflect/qt/QmlStreamer.cpp

```diff
@@ -66,6 +66,11 @@ QQmlEngine* QmlStreamer::getQmlEngine()
     return _impl->getQmlEngine();
 }
 
+Stream *deflect::qt::QmlStreamer::getStream()
```

Comment:
Ok then :) if we want to make it further complicated, most reasonable will be having a REST interface in DC and sending REST requests from qml ;) Almost no C++ afterwards.